### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by tony**

### CODE REVIEW

**Comment by hellysmile** [comment]
Can we get this geen in reafactoring branch?

**Comment by tony** [comment]
If we merge this into master, I'll merge it into / rebase it into #17

**Comment by hellysmile** [comment]
Thete is some missing statemets regarding django-extension
Also isort checks is missed

**Comment by hellysmile** [comment]
You are welcome to merge it in refactoring directly

**Comment by tony** [comment]
> There is some missing statemets regarding django-extension

Can you explain this?

File: tox.ini [hunk 1]
```diff
@@ -1,87 +1,45 @@
...
-    flake8
-    flake8-blind-except
-    flake8-debugger
-    flake8-print
-    flake8-quotes
-    pep8-naming
-commands =
-    flake8 --show-source django_activeurl
+	flake8
+commands = flake8 django_extensions tests
```
Comment (hellysmile) [inline tox.ini]: Here is wrong package name statement

**Comment by hellysmile** [comment]
What is safity and precommit?

**Comment by tony** [comment]
Removed remnants of precommit, but safety is https://pypi.python.org/pypi/safety/1.5.1 (https://github.com/pyupio/safety)

Want to keep or remove?

**Comment by hellysmile** [comment]
Lets merge this to the master when it will becomes green

**Comment by hellysmile** [comment]
As for me we can remove safity

**Comment by tony** [comment]
Removed safety

**Comment by tony** [comment]
Also, I slipped in a change from nose -> pytest in 07595ff

**Pull request merged**